x-pack/plugin/sql/qa/server/security/src/test/java/org/elasticsearch/xpack/sql/qa/security/RestSqlSecurityIT.java

```diff
@@ -281,18 +281,27 @@ protected AuditLogAsserter createAuditLogAsserter() {
     }
 
     /**
-     * Test the hijacking a scroll fails. This test is only implemented for
-     * REST because it is the only API where it is simple to hijack a scroll.
+     * Test the hijacking a cursor fails. This test is only implemented for
+     * REST because it is the only API where it is simple to hijack a cursor.
      * It should exercise the same code as the other APIs but if we were truly
      * paranoid we'd hack together something to test the others as well.
      */
-    public void testHijackScrollFails() throws Exception {
-        createUser("full_access", "rest_minimal");
+    public void testHijackCursorFails() throws Exception {
```

Comment:
Previously, this test asserted that a user with sufficient privilege to perform an equivalent request cannot hijack someone else's cursors. Since scroll cursors are "owned" by users, this was something SQL guaranteed for search hit queries. Now, a PIT is shared between users and this guarantee no longer holds.

The test now asserts that users with less privileges cannot hijack cursors for all sorts of queries.

Reply:
It doesn't test anything specific to PIT it just ensures that SQL cursors do not magically bypass ES security (which is very unlikely I hope).